spec/content_metadata_spec.rb

```diff
@@ -616,30 +616,56 @@
       end
     end
 
-    context 'when using a 3d object with one 3d type files and three other supporting files (where one supporting file is a non-viewable but downloadable 3d file)' do
+    context 'when using a 3d object with one 3d type files and three other supporting files (where two supporting files are non-viewable but downloadable 3d files)' do
```

Comment:
This test seems confusing to me because the outputs "jpegs are shelved" are not dependent on the inputs "when using a 3d object".  The test leads you to believe that this behavior is dependent on it being a 3d object.

Reply:
Yes, technically this is just verifying that a 3d-like object works out ok, but it is true that all the shelve/publish/preserve attributes are based purely on mimetype and not object-type.  I could clarify this in a comment?